Doc/library/ssl.rst

```diff
@@ -1328,12 +1328,12 @@ SSL sockets also have the following additional methods and attributes:
    If any precondition isn't met (e.g. not TLS 1.3, PHA not enabled), an
    :exc:`SSLError` is raised.
 
-   .. versionadded:: 3.7.1
-
    .. note::
       Only available with OpenSSL 1.1.1 and TLS 1.3 enabled. Without TLS 1.3
       support, the method raises :exc:`NotImplementedError`.
 
+   .. versionadded:: 3.7.1
```

Comment:
hum wait, why is it version 3.8 in the master branch but 3.7.1 in the 3.7 branch? The versionadded should be the same in 3.7 and master branches, no? cc @csabella @tiran (I only checked verify_client_post_handshake, I didn't check other versionadded markups.)

Reply:
That's how it was when originally added in GH-9460 and GH-9505, so I didn't change it.  If it should be 3.7.1 on both, I'll update master.